torchdynamo/skipfiles.py

```diff
@@ -35,10 +38,12 @@
     for m in (
         abc,
         collections,
+        contextlib,
         copy,
         copyreg,
         dataclasses,
         enum,
+        functools,
```

Comment:
This should go in the list below with the try/except (so we don't error if it is not installed).

Reply:
Done.